libsweep/src/sweep.cc

```diff
@@ -19,25 +22,17 @@ typedef struct sweep_device {
 
 #define SWEEP_MAX_SAMPLES 4096
 
-typedef struct sweep_scan {
+struct sweep_scan {
```

Comment:
Actually I think I'll just leave it for now and provide another PR once / if this one gets merged